anastruct/__init__.py

```diff
@@ -1,3 +1,3 @@
 from anastruct.fem.system import SystemElements
 from anastruct.fem.util.load import LoadCase, LoadCombination
-from anastruct.vertex import Vertex
```

Comment:
Could you drop this change?

Reply:
Sure. Sorry I did that, I'm new in Github. I'll close this PR and open a new one.